imap/caldav_alarm.c

```diff
@@ -1783,7 +1783,7 @@ static void process_one_record(struct caldav_alarm_data *data, time_t runtime, i
 #ifdef WITH_JMAP
     else if (mbtype_isa(mailbox_mbtype(mailbox)) == MBTYPE_JMAPSUBMIT) {
         if (record.internaldate > runtime || dryrun) {
-            caldav_alarm_bump_nextcheck(data, record.internaldate, runtime, error_message(r));
+            caldav_alarm_bump_nextcheck(data, record.internaldate, runtime, NULL);
```

Comment:
Is this OK to be running a replace after a past error, or will that wipe the error?

(yeah, I know I'm being annoying, but I'm not sure the various orders in which this can happen)

Reply:
@brong If we pass a null err_str to bump_nextcheck, we use the current data->err_str in the replace.  Same if last_run is 0 (zero).

But, your comment highlights an issue with the above line in that I should be passing last_run as 0 so we don't change it, since we haven't actually tried to send the message here.